app/lib/feature_flags.rb

```diff
@@ -0,0 +1,5 @@
+class FeatureFlags
```

Comment:
Big fan of adding this so we're not accessing env vars all over the place (and so there's a place to see what the feature flags currently are).